packages/legend-graph/src/models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_ValueSpecificationSerializer.ts

```diff
@@ -320,7 +320,7 @@ const runtimeInstanceModelSchema = createModelSchema(V1_RuntimeInstance, {
   _type: usingConstantValueSchema(V1_ValueSpecificationType.RUNTIME_INSTANCE),
   runtime: custom(
     (val) => V1_serializeRuntime(val),
-    (val) => deserialize(V1_engineRuntimeModelSchema, val),
+    (val) => deserialize(V1_engineRuntimeModelSchema(), val),
```

Comment:
Since effect of `createModelSchema` is global, you don't need to do this, do `deserialize(V1_EngineRuntime, val)` instead